crates/next-core/src/next_client/context.rs

```diff
@@ -81,6 +82,22 @@ pub fn get_client_resolve_options_context(
         resolved_map: Some(next_client_resolved_map),
         browser: true,
         module: true,
+        rules: vec![(
+            ContextCondition::InDirectory("node_modules".to_string()),
+            Some(
+                ResolveOptionsContext {
+                    enable_node_modules: true,
+                    custom_conditions: vec!["development".to_string()],
+                    import_map: Some(next_client_import_map),
+                    fallback_import_map: Some(next_client_fallback_import_map),
+                    resolved_map: Some(next_client_resolved_map),
+                    browser: true,
+                    module: true,
+                    ..Default::default()
+                }
+                .cell(),
+            ),
+        )],
```

Comment:
This should be:

```rust
let shared_options = ResolveOptionsContext {
    enable_node_modules: true,
    custom_conditions: vec!["development".to_string()],
    import_map: Some(next_client_import_map),
    fallback_import_map: Some(next_client_fallback_import_map),
    resolved_map: Some(next_client_resolved_map),
    browser: true,
    module: true,
    ..Default::default()
};
```

Then spread that into both this and the one above. This way, if we ever add new keys in the future, they will be applied to both correctly.

The same remark applies to all of the below.